google-cloud-language/README.md

```diff
@@ -1,72 +1,48 @@
-# google-cloud-language
+# Ruby Client for Google Cloud Natural Language API ([Beta](https://github.com/GoogleCloudPlatform/google-cloud-ruby#versioning))
 
-[Google Cloud Natural Language API](https://cloud.google.com/natural-language/) ([docs](https://cloud.google.com/natural-language/docs)) reveals the structure and meaning of text by offering powerful machine learning models in an easy to use REST API. You can use it to extract information about people, places, events and much more, mentioned in text documents, news articles or blog posts. You can use it to understand sentiment about your product on social media or parse intent from customer conversations happening in a call center or a messaging app. You can analyze text uploaded in your request or integrate with your document storage on Google Cloud Storage.
+[Google Cloud Natural Language API][Product Documentation]:
+Google Cloud Natural Language API provides natural language understanding
+technologies to developers. Examples include sentiment analysis, entity
+recognition, and text annotations.
+- [Client Library Documentation][]
+- [Product Documentation][]
 
-The Cloud Natural Language API currently supports English, Spanish, and Japanese for sentiment analysis, entity analysis, and syntax analysis.
-
-- [google-cloud-language API documentation](http://googlecloudplatform.github.io/google-cloud-ruby/#/docs/google-cloud-language/latest)
-- [google-cloud-language on RubyGems](https://rubygems.org/gems/google-cloud-language)
-- [Google Cloud Natural Language API documentation](https://cloud.google.com/natural-language/docs)
+*This release, 0.28.0, introduces breaking changes relative to the previous
+release, 0.27.1. For more details and instructions to migrate your code, please
+visit the [migration guide](https://cloud.google.com/natural-language/docs/ruby-client-migration).*
```

Comment:
I'm getting a 404 for the migration guide URL.

Reply:
Yes, that's intentional. It cannot go live until samples are published, and samples cannot be published until the gem is published... We'll coordinate (hopefully tomorrow) with @frankyn to ensure there's a minimal amount of time during which this link is broken.

Reply:
That's goal! Minimize the delay for that.